PennMobile/Course Alerts/Networking/CourseAlertNetworkManager.swift

```diff
@@ -158,6 +159,29 @@ class CourseAlertNetworkManager: NSObject, Requestable {
         }
     }
 
+    func updatePathRegistration(srcdb: String, crns: [String], callback: @escaping (_ success: Bool, _ error: Error?) -> Void) {
```

Comment:
Is it possible to make this an async method to make PathAtPennNetworkManager a bit cleaner? We can do this by moving the `withCheckedThrowingContinuation` inside here.